### ISSUE TITLE
Play Store hosting

### ISSUE THREAD

**Issue by sidhshar** [S3Infosoft/s3-loyalty-mobile#11]
Plan the steps to host on Play Store

**Comment by dev-sourabh** [comment]
https://play.google.com/apps/publish/

**Comment by sidhshar** [comment]
Sent Invite to you and Varun for Google Play Console access

**Comment by Varun-L** [comment]
sir should we publish the app to playstore. ? Me n Sourabh are confused , we are interested to know why you gave us the Console Access on Google Play

**Comment by Varun-L** [comment]
After the App is Complete publishing is very simple sir , It takes 5 mins to finish the process. All we need is App Icon or Logo and a Bunch of ScreenShots and a Description of our App. Post the Completion of our process it takes 1 hour 15 mins around to get published. after we upload it checks the app and after verification by Google it will get available after 1 hour.

**Comment by sidhshar** [comment]
Yes, we are not publishing the app currently until further notice. The Console Access is just in case if it was required by you for any activities.
